packages/testing/containers/n8n-test-container-creation.ts

```diff
@@ -98,22 +99,22 @@ export async function createN8NStack(config: N8NConfig = {}): Promise<N8NStack>
 	const { postgres = false, queueMode = false, env = {}, projectName } = config;
 	const queueConfig = normalizeQueueConfig(queueMode);
 	const usePostgres = postgres || !!queueConfig;
-	const uniqueProjectName = projectName ?? `n8n-${Math.random().toString(36).substring(7)}`;
+	const uniqueProjectName = projectName ?? `n8n-stack-${Math.random().toString(36).substring(7)}`;
 	const containers: StartedTestContainer[] = [];
-	let network: StartedNetwork | undefined;
-	let nginxContainer: StartedTestContainer | undefined;
 
-	let environment: Record<string, string> = { ...BASE_ENV, ...env };
+	const network = await new Network().start();
 
-	if (usePostgres || queueConfig) {
-		network = await new Network().start();
-	}
+	let environment: Record<string, string> = {
+		...BASE_ENV,
+		...env,
+		N8N_PROXY_HOPS: '1',
+	};
 
 	if (usePostgres) {
 		const postgresContainer = await setupPostgres({
 			postgresImage: POSTGRES_IMAGE,
 			projectName: uniqueProjectName,
-			network: network!,
+			network: network,
```

Comment:
Do we have linting not setup in this package? This could be
```suggestion
			network,
```

Reply:
We do now, thanks for catching that!